### DISCUSSION TITLE
bpftrace v0.14 all programs fail with Segmentation fault

### DISCUSSION THREAD

**Discussion by godjiang2019** [bpftrace/bpftrace#3026] [Q&A] [answered]
I executed 'apt install -y bpftrace' to install, but all programs are failing with a Segmentation Fault. What should I do to solve this issue?
![image](https://github.com/bpftrace/bpftrace/assets/48830951/0533fe3c-4ee5-4b00-9881-75629417dc2a)
![image](https://github.com/bpftrace/bpftrace/assets/48830951/92a44ccb-1a37-44af-acc2-7f044e0aeb2a)
![image](https://github.com/bpftrace/bpftrace/assets/48830951/7f851046-76ab-481e-97cb-7df0d2f21d9b)

**Comment by danobi** [comment, marked as answer]
Hi, thanks for the report. v0.14 was released almost 3 years ago and since then we've probably fixed thousands of bugs. Can you try again on the latest build? See https://github.com/bpftrace/bpftrace/blob/master/INSTALL.md#appimage-install for one easy way to get the latest.

If the segfault still occurs then we'd be happy to help debug further.

**Reply by godjiang2019** [reply]
Thank you for your response, I have resolved my issue.
